github/repos_collaborators.go

```diff
@@ -16,7 +16,8 @@ type ListCollaboratorsOptions struct {
 	// Affiliation specifies how collaborators should be filtered by their affiliation.
 	// Possible values are:
 	//     outside - All outside collaborators of an organization-owned repository
-	//     direct - All collaborators with permissions to an organization-owned repository, regardless of organization membership status
+	//     direct - All collaborators with permissions to an organization-owned repository,
+	//     regardless of organization membership status
```

Comment:
Did you do this differently than I suggested on purpose, or was it unintentional?

I still think the version I suggested looks a bit nicer, and it's definitely more consistent with the rest of the repo (see "precedent" snippet I included).

Reply:
It was unintentional - I'm sorry for that mistake on my part.

Reply:
No worries, thanks for clearing it up!